python/RappCloud/CloudMsgs/ObjectDetectionLearnObject.py

```diff
@@ -0,0 +1,79 @@
+from RappCloud.Objects import (
+    File,
+    Payload)
+
+from Cloud import (
+    CloudMsg,
+    CloudRequest,
+    CloudResponse)
+
+
+class ObjectDetectionLearnObject(CloudMsg):
+    """ Object Detection - Learn Object CloudMsg object"""
+
+    class Request(CloudRequest):
+        """ Object Detection - Learn Object Cloud Request object. ObjectDetectionLearnObject.Request """
+        def __init__(self, **kwargs):
+            """!
+            Constructor
+
+            @param **kwargs - Keyword arguments. Apply values to the request attributes.
+                - @ref fname
+                - @ref name
+                - @ref user
+            """
+
+            ## File path to the image to load. This is the sample object image.
+            self.fname = ''
+            ## Model name
+            self.name = ''
+            ## Username 
+            self.user = 'rapp'
```

Comment:
To remove. User information is retrieved on the RAPP Platform using the user's credentials (token).
Resource [here](https://github.com/rapp-project/rapp-platform/wiki/How-to-create-a-HOP-service-for-a-ROS-service%3F#web-service-implementation)